tests/spec.lua

```diff
@@ -22,14 +22,35 @@ local function newFolder(name, parent, content)
 	return folder
 end
 
+-- polyfill for table.create - lemur doesn't support this yet.
+function table.create(size, value)
+	local t = {}
+	for i = 1, size do
+		t[i] = value or nil
```

Comment:
There shouldn't be an `or nil` here. All that would do is change `value` from `false` to `nil`.

Reply:
Good point lol, not sure why I put that there.